packages/lumx-core/src/scss/components/text/_index.scss

```diff
@@ -3,23 +3,26 @@
    ========================================================================== */
 
 .#{$lumx-base-prefix}-text {
+    white-space: var(--lumx-text-white-space);
+
     &--is-truncated {
+        --lumx-text-white-space: nowrap;
+
         display: block;
         overflow: hidden;
         text-overflow: ellipsis;
-        white-space: nowrap;
     }
 
     &--is-truncated-multiline {
         /* autoprefixer: off */
-        display: -webkit-box;
+        display: box;
```

Comment:
This seems to break multiline truncate (tested on chrome on SB https://5fbfb1d508c0520021560f10-knwwndcxte.chromatic.com/?path=/story/lumx-components-text-text--truncate-multiline)